py_test.py

```
@@ -0,0 +1,374 @@
+                ScrapOrder = self.env['stock.scrap']
+                scrap_domain = [
+                    ('raw_mrp_id', 'in', self.ids),
+                    ('product_id', '=', move0.product_id.id),
+                    ('lot_id', 'in', move0.lot_ids.ids),
+                ]
+                scrap_orders = ScrapOrder.search(scrap_domain)
+                scrap_qty = sum(scrap.scrap_qty for scrap in scrap_orders)
+                move_qty = move0.product_uom_qty - scrap_qty
+                if move_qty < 0:
+                    move_qty = 0  # avoid negative move
+                move0.product_uom_qty=move_qty
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)